packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -141,8 +142,15 @@ TextStyle primaryColorLight(TextStyle style, BuildContext context) {
 /// Builds an [ErrorReporter] for a context that shows a [SnackBar].
 ErrorReporter showErrorSnackBar(BuildContext context) {
   return (String title, dynamic error) {
-    Scaffold.of(context).showSnackBar(SnackBar(
-      content: Text(title),
-    ));
+    // TODO(kenz): this isn't the cleanest - we should fix the issue in a better
+    // way.
```

Comment:
can you file a bug? I don't quite understand how we are calling this at a time when the Element tree isn't valid.